dataset/src/main/scala/frameless/ops/Repeat.scala

```diff
@@ -0,0 +1,26 @@
+package frameless
+package ops
+
+import shapeless.{HList, Nat, Succ}
+import shapeless.ops.hlist.Prepend
+
+/* By Jeremy Smith */
```

Comment:
/** Typeclass supporting repeating L-typed HLists N times.
      *  
      * Repeat[Int :: String :: HNil, Nat._2].Out =:=
      * Int :: String :: Int :: String :: HNil
      *
      * By Jeremy Smith. To be replaced by `shapeless.ops.hlists.Repeat`
      * once (https://github.com/milessabin/shapeless/pull/730 is published.
      */